### PR TITLE
sample applications and grafana-operator argocd app for multi-account CDK Observability accelerator

### PR DESCRIPTION
**Pull request by iamprakkie**
This PR adds three new contents to artifacts:
1. [grafana-operator-chart](https://github.com/iamprakkie/aws-observability-accelerator/tree/artifacts/artifacts/grafana-operator-chart) - helm charts to deploy `grafana-operator` resources like datasources, dashboards and grafana identity.
2. [sample-apps](https://github.com/iamprakkie/aws-observability-accelerator/tree/artifacts/artifacts/sample-apps) - sample argocd apps that will be deployed multi-account clusters and [grafana-operator-app](https://github.com/iamprakkie/aws-observability-accelerator/tree/artifacts/artifacts/sample-apps/grafana-operator-app) to deploy grafana-operator resources as argocd apps. 
3. [teams](https://github.com/iamprakkie/aws-observability-accelerator/tree/artifacts/artifacts/teams) - supporting resource for sample argocd apps.

### CODE REVIEW

**Review by elamaran11 — APPROVED** [review]
LGTM. @bonclay7 Can you review this. We are adding new setup of artifacts to support multi account pattern with Argo CD - new flavour which our customers ask

**Comment by elamaran11** [comment]
@iamprakkie Please fix the warnings.

**Comment by iamprakkie** [comment]
@elamaran11 All warnings fixed

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@iamprakkie Looks good, have below comments.
1. See too many files checkin, can you check in files only related to the multi account/region pattern?
2. Remove unnecessary stuff like teams, karpenter. Lets keep the PR slim as far as possible
3. Can we group all Argo artifacts under Argo folder?

**Comment by iamprakkie** [comment]
incorporated review comments

1. moved teams, sample-apps and grafana-operator-chart to artifacts/argocd-apps
2. removed teams not in use with multi-account pattern (removed team-kirk, team-data, team-danger, team-scan, team-platform, team-burnham and team-riker). retained team-geordie and team-carmen
3. moved grafana-operator-app to artifacts/argocd-apps
4. removed samples-apps/envs/dev and samples-apps/envs/test. Retaining only samples-apps/envs/prod
5. updated apps with path changes

File: artifacts/argocd-apps/sample-apps/envs/prod/templates/team-carmen.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
```
Comment (elamaran11) [inline artifacts/argocd-apps/sample-apps/envs/prod/templates/team-carmen.yaml:2]: Why is Team Carmen needed? he is my skip level btw.

File: artifacts/argocd-apps/grafana-operator-chart/templates/amg_grafana-amp-datasource.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
+metadata:
+  name: grafanadatasource-amp
+  namespace: grafana-operator
+spec:
```
Comment (elamaran11) [inline artifacts/argocd-apps/grafana-operator-chart/templates/amg_grafana-amp-datasource.yaml:6]: These file - [https://github.com/aws-observability/aws-observability-accelerator/pull/28/files#diff-f0c4fb63114ba78a5db50b55191[…]d8d51abe9b7ff3a5abcc9cd2d375a2f8](https://github.com/aws-observability/aws-observability-accelerator/pull/28/files#diff-f0c4fb63114ba78a5db50b5519126758d8d51abe9b7ff3a5abcc9cd2d375a2f8) Can we not use the original files in GitOps repo which we use for flux

like this [https://github.com/aws-observability/aws-observability-accelerator/blob/main/artif[…]or-manifests/eks/infrastructure/amg_grafana-amp-datasource.yaml](https://github.com/aws-observability/aws-observability-accelerator/blob/main/artifacts/grafana-operator-manifests/eks/infrastructure/amg_grafana-amp-datasource.yaml)

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@iamprakkie Have some feedback

File: artifacts/argocd-apps/grafana-operator-chart/templates/amg_grafana-amp-datasource.yaml (hunk 2 above)
Comment (iamprakkie) [reply artifacts/argocd-apps/grafana-operator-chart/templates/amg_grafana-amp-datasource.yaml:6]: Difference b/w existing one and new one is, new one is compatible for ArgoCD based on values.. whereas old one is compatible with FluxCD kustomization.

File: artifacts/argocd-apps/sample-apps/envs/prod/templates/team-carmen.yaml (hunk 1 above)
Comment (iamprakkie) [reply artifacts/argocd-apps/sample-apps/envs/prod/templates/team-carmen.yaml]: team-carmen removed

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**